custom_components/spotcast/helpers.py

```diff
@@ -171,6 +171,28 @@ def get_random_playlist_from_category(spotify_client:spotipy.Spotify, category:s
 
     return chosen['uri']
 
+
+def url_to_spotify_uri(url: str) -> str:
```

Comment:
do not return None on error, Raise an error instead to let the parent process what to do on error

Reply:
Sure, makes sense. What's the convention around error types for the project?